src/constants/game-constants.ts

```diff
@@ -24,6 +24,11 @@ export const IV_MIN = 0;
  */
 export const IV_MAX = 31;
 
+/**
+ * Value used for the IVs of starters unlocked by default on a fresh file.
+ */
+export const IV_DEFAULT = 15;
```

Comment:
I think this constants name should be more specialized that it's only for `starters`.
Should also be the actual array I'd say. Maybe we want to alter specific IV aspects in the future and not the whole array.

```suggestion
export const IV_DEFAULT_UNLOCKED_STARTERS = [15, 15, 15, 15, 15, 15];
```

Reply:
I think I'd go with `DEFAULT_STARTER_IVS`.

Reply:
Agree about the name, I was just trying to match the existing constants. Will update tomorrow
Unsure about storing the whole array though.
After having looked at the heap so much I think we need to be a bit more mindful about the amount of data we keep in memory at all times. Sure that single array won't make much of a difference but I don't know about keeping an array that is only ever relevant the very first time the game is ran for a potential future use that is very unlikely to happen